tests/utils/utils.py

```diff
@@ -0,0 +1,8 @@
+from contextlib import contextmanager
+
+
+@contextmanager
```

Comment:
shouldn't that be within the tests part?

Reply:
Indeed, I moved it to `tests`. I didn't realize it could bring confusion with the actual testing of the `utils` package.